### ISSUE TITLE
[FR] LDAPS support for the official Taiga6 docker installation

### ISSUE THREAD

**Issue by nuokh** [taigaio/taiga-docker#19]
**TLDR: Please add official LDAPS support to the Taiga 6 docker installation and/or provide some step by step guide explaining how to install and maintain the community addons to the docker installation. That would be great.**

Okay so please excuse that I have to get some tears flowing regarding the missing (suported) LDAPS functionality in Taiga 6: 

We're currently evaluating Taiga for a non profit hospital. We have absolutely zero budget to buy ourself project management software and using free cloud services or any cloud services at all is forbidden by german law. We also have close to zero time allocated to improving the status quo of working with excel files on network shares so we're currently struggling through getting Taiga 6 production ready in our free time in the middle of already pretty high workloads due to obvious reasons.

Taiga looks like a gift from heaven, enabling us to finally move away from excel files on network shares to something visually and functionally appealing (even for non-IT teams. Looking at you, free JIRA software license for nonprofits where the software focused terminology and workflow is forced upon everybody…) to coordinate tasks and ideas across teams from different departments.

Okay so now that I built up the appropriate amount of pity to get your attention :-P

Please please please take us on-premise folks that have to rely heavily on LDAPS into account when planning your next development sprints. I am pretty sure most of us on premise folks have tons of experience with virtualization solutions like VMWare but are total docker noobs.

I know that I am struggling through a lot of problems that are basically solved with basic docker knowledge and installing the community plugin in your docker installation is probably easy for a dev with a lot of Docker experience but for us old and gnarly sysadmins it's just super confusing and we have to make sure every deployment checks certain boxes like “can we test backups, updates, restores” and “can we plug it into our LDAP to manage our 300 users that constantly forget passwords”.

Thanks for the great work overall, this project really looks like it could shape up to become the gold standard in free and open source project management :-)

**Comment by kota65535** [comment]
Any progress?

**Comment by TuringTux** [comment]
I just took a day or so to figure out how to integrate an LDAP contrib plugin with the Taiga Docker setup. I tried to document my efforts in the community forum:

https://community.taiga.io/t/integrate-an-ldap-account-database-with-taiga/212

I am not sure if this will help you anymore (you've probably figured it out by now), but maybe it can be helpful to someone in the future.
